### DISCUSSION TITLE
SU SEED Labs - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#512] [Announcements]
# SU SEED Labs - CS自学指南

CS自学指南

https://csdiy.wiki/%E7%B3%BB%E7%BB%9F%E5%AE%89%E5%85%A8/SEEDLabs/

**Comment by TephrocactusMYC** [comment]
这个模块不行专门分出来一个CTF模块得了。

**Reply by ZhuoHanWang** [reply]
网安有标准的教科书都少，课程也很少，我本科网安，大一学了寂寞，大二想打比赛了，干脆学数据结构打程序设计比赛发育了，ctf打不明白真的

**Reply by TephrocactusMYC** [reply]
我不太理解你回复我的内容是什么意思。这里面包括pwn college在内有太多内容属于CTF而不是系统安全。我提议专门分出来一个叫做CTF的模块。

**Reply by wgggggggg** [reply]
大佬可以推荐一下着重于CTF的一些课程吗

**Comment by LCJD99** [comment]
## 找同学一起学习
计划后续还会学习其他课程(目前是 **cs61b** 和 **本课程**)。

创建了一个qq学习群，有一起学习的同学可以加群一起讨论，qq群： **563729367**

**Comment by Shaddock42125** [comment]
感谢
